src/layer/road.js

```diff
@@ -709,8 +733,11 @@ class Service extends Road {
 
     this.constraints = [
       "all",
-      ["!=", "toll", 1],
-      ["!in", "service", "parking_aisle", "driveway"],
+      ["!=", ["get", "toll"], 1],
+      [
+        "!",
+        ["in", ["get", "service"], ["literal", ["parking_aisle", "driveway"]]],
```

Comment:
Use a `match` expression that evaluates to 0 if it’s one of the cases and 1 otherwise.